### ISSUE TITLE
prepare_wham_input does not use several values from ASAP3 data file

### ISSUE THREAD

**Issue by brianstock-NOAA** [timjmiller/wham#23]
ASAP3 data files have an entry for which ages to average F over to report total F, e.g.
```
# Age Range for Average F
3  5
```

Line 474 of `prepare_wham_input` does not use this:
```
data$Fbar_ages = 1:data$n_ages
```

**Comment by brianstock-NOAA** [comment]
Fixed in 70a698ef4b96abacb9043ddc930066b7711f51de. NAA in first year (`input$par$log_N1_pars`), F in first year (`input$par$log_F1`), and q (`input$par$logit_q`) are now initialized at values from asap3 file. Ages to average F over (`input$data$Fbar_ages`) is also now taken from asap3 file.

**Comment by brianstock-NOAA** [comment]
Also ce98e17971fa68738edba140145fa99d14256e70: if using mean recruitment (no SR function), initialize `R0` at initial NAA-1.
